src/dogear.js

```diff
@@ -5,141 +5,133 @@ const Oppsy = require('oppsy');
 const Hoek = require('hoek');
 const At = require('lodash.at');
 
-const internals = {
-  defaults: {
-    statsdConfig: {
-      prefix: 'hapi.'
-    },
-    opsInterval: 1000,
-    opsMetrics: null
+const defaultConfig = {
```

Comment:
This is a non-functional styling change that adds unnecessarily to the diff. There is no perceivable benefit to this change. Please revert and limit changes to the scope of the change, and not introduce different structure and styling without functional benefit.

Reply:
`hapi` did change their default code style, but I prefer that PRs are limited to correcting only the issue at hand. The current style of this file will still work with newer versions of `hapi`, thus no pressing reason to change it now.